docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -57,7 +57,7 @@ docker run \
   {synthetics-image} -e \
 ----
 
-NOTE: You may need to set additional environment variables. See the https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[{agent} environment variables guide] for more information.
+NOTE: You may need to set additional environment variables. See the https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[{agent} environment variables guide] for more information. See https://www.elastic.co/guide/en/fleet/master/elastic-agent-container.html[Run {agent} in a container] for more information on running {agent} with Docker.
```

Comment:
```suggestion
NOTE: You may need to set additional environment variables. See the https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[{agent} environment variables guide] for more information. See {fleet-guide}/elastic-agent-container.html[Run {agent} in a container] for more information on running {agent} with Docker.
```

Reply:
We need to use variables to resolve link paths so that the docs use the correct branch when we backport.